cmd/vcert/commands.go

```diff
@@ -397,11 +592,22 @@ func doCommandCredMgmt1(c *cli.Context) error {
 				fmt.Println("refresh_token: ", resp.Refresh_token)
 			}
 		} else if cfg.Credentials.User != "" && cfg.Credentials.Password != "" {
-			resp, err := tppConnector.GetRefreshToken(&endpoint.Authentication{
+
+			auth := &endpoint.Authentication{
 				User:     cfg.Credentials.User,
 				Password: cfg.Credentials.Password,
 				Scope:    flags.scope,
-				ClientId: flags.clientId})
+				ClientId: flags.clientId}
+
+			if flags.sshCred {
+				auth.Scope = "ssh:manage"
+			} else if flags.pmCred {
+				auth.Scope = "configuration:manage"
```

Comment:
@angelmoo I think this was my fault for telling you wrong... I think policy management requires `"certificate:manage"` for `getpolicy` and `"configuration:manage"` for `setpolicy`, right?  If so, we should get rid of the `--cpm` flag and update the scope for the `--pm` flag to `"certificate:manage,revoke;configuration:manage"`.

Reply:
yes, and fixed in most recent commit